docs/docs/en/guide/installation/rainbond.md

```diff
@@ -0,0 +1,64 @@
+# Use Rainbond Deployment
+
+This article describes the one-click deployment of highly available DolphinScheduler clusters through the [Rainbond](https://www.rainbond.com/) cloud native application management platform,This method is suitable for users who don't know much about complex technologies such as Kubernetes,Lowered the threshold for deploying DolphinScheduler in Kubernetes.
+
+## Prerequisites
+
+* Available Rainbond cloud native application management platform，Please refer to the documentation [Rainbond Quick install](https://www.rainbond.com/docs/quick-start/quick-install)
+
+## DolphinScheduler Cluster One-click Deployment 
+
+* Docking and accessing the built-in open source app store,Search the keyword `dolphinscheduler` to find App DolphinScheduler.
```

Comment:
May I ask what does `docking and accessing the built-in open source app store` mean? Is the app store in `Rainbond`?

Reply:
> May I ask what `docking and accessing the built-in open source app store` means here? Is the app store in `Rainbond`?

Rainbond app store is saas service,need to register users,so need authorization in builtin open source app store

Reply:
> > May I ask what `docking and accessing the built-in open source app store` means here? Is the app store in `Rainbond`?
> 
> Rainbond app store is saas service,need to register users,so need authorization in builtin open source app store

I got it. Thx for the explanation : )